samples/SampleApp/Program.fs

```diff
@@ -1,41 +1,84 @@
 ﻿open Farmer
```

Comment:
Can you undo changes to this file and instead add a sample script? I'm at fault for committing to this as well - it's useful as a quick scratchpad but not more.